physionet-django/console/templates/console/training_process.html

```diff
@@ -48,9 +45,34 @@ <h5>Training</h5>
                   {% if training.completion_report_url %}
                     <li>URL: <mark><a href="{{ training.completion_report_url }}" target="_blank">External link</a></mark></li>
                   {% endif %}
+                  <li>Submitted: {{ training.application_datetime|date }}</li>
                 </ul>
             </div>
         </div>
+        <div class="card mb-4">
+            {% if parsed_training_pdf.unknown_cert_type %}
```

Comment:
I can't see this `unknown_cert_type` thing being set anywhere. From what I see `parsed_training_pdf` is simply a dict.